contracts/forwarder/Forwarder.sol

```diff
@@ -60,12 +60,16 @@ contract Forwarder is IForwarder {
 
         bytes memory callData = abi.encodePacked(req.data, req.from);
         require( gasleft()*63/64 >= req.gas, "FWD: insufficient gas" );
+        // solhint-disable-next-line not-rely-on-time
+        require(req.validUntil==0 || req.validUntil > block.timestamp, "FWD: request expired");
```

Comment:
Also, we have specifically agreed to use block numbers, not timestamps, for validity. As well as everything else as we have agreed with the rule of not relying on block time. 
What changed since?

Reply:
Also, 'block' should really appear in the field's name

Reply:
may I suggest 'expirationBlock' (or 'validUntilBlock', but that is 3 words)